src/Stormpath/Oauth/CreateGrantAuthenticationAttempt.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+namespace Stormpath\Oauth;
+
+
+use Stormpath\Resource\AccountStore;
+use Stormpath\Resource\Resource;
+
+class CreateGrantAuthenticationAttempt extends Resource
```

Comment:
I just talked to @maranai and sounds like she will be renaming some classes, including this one. So maybe sync up with her to see what the final naming convention will be.